manufacturing-ai-suite/wind-turbine-anomaly-detection/helm/templates/time-series-analytics-microservice.yaml

```diff
@@ -10,9 +10,14 @@ metadata:
   name: {{ .Values.config.time_series_analytics_microservice.name }}
   namespace: {{ .Values.namespace }}
 spec:
+  type: NodePort
   ports:
   - port: {{ .Values.config.time_series_analytics_microservice.kapacitor_port }}
     name: kapacitor-port
```

Comment:
Are we using this kapacitor port anywhere? maybe needed by telegraf to connect to?